apps/dashboard/src/components/auth/usecase-selector.tsx

```diff
@@ -0,0 +1,74 @@
+import { ArrowLeft } from 'lucide-react';
+import { Card, CardContent } from '../primitives/card';
+import { ChannelTypeEnum } from '@novu/shared';
+import { Usecase } from './usecases-list.utils';
+
+interface UsecaseSelectOnboardingProps {
+  onHover: (id: ChannelTypeEnum | null) => void;
+  onClick: (id: ChannelTypeEnum) => void;
+  selectedUseCases: ChannelTypeEnum[];
+  channelOptions: Usecase[];
+}
+
+export function UsecaseSelectOnboarding({
+  onHover,
+  onClick,
+  selectedUseCases,
+  channelOptions,
+}: UsecaseSelectOnboardingProps) {
+  return (
+    <div className="flex w-full flex-col items-center justify-center">
+      <div className="flex w-full flex-col items-center gap-8">
+        <div className="flex w-full flex-col items-start gap-1">
```

Comment:
these 3 divs could be simplified to one with these children: header, channel cards, footer. The gap between them seems to be the same.